Receiver/simpleMovingAverage.py

```diff
@@ -0,0 +1,31 @@
+def calculateAvg(list_temp,list_soc):
+    final_display_list = {}
+    list_temp.sort()
+    list_soc.sort()
+    final_display_list['temperature'] = list_temp
+    final_display_list['soc'] =list_soc
+    print(final_display_list)
+    displayOutput(final_display_list)
+    
+def calculateMin(bms_data,param):
```

Comment:
The file is called `simpleMovingAverage` but has the calculation of other statistics also. Would `computeStats` have been a better name?